### ISSUE TITLE
Bug when deleting a note

### ISSUE THREAD

**Issue by Kitanit** [nextcloud/notes#907]
next 24.0.3
latest version of notes

![image](https://user-images.githubusercontent.com/65107731/187147415-ab2a766d-e0b8-4095-ad3c-3e581f71c7be.png)

**Comment by stefan-niedermann** [comment]
Indeed, looks like a typo (`tilte`), so the property binding fails.

**Comment by korelstar** [comment]
Thanks for reporting this. This app uses [Transifex](https://www.transifex.com/nextcloud/nextcloud/notes/) for translations. Hence, this error has to be fixed in the translation over there. You could create an account over there and help with translations.

But for this specific issue, maybe @rakekniven could also help?

**Comment by korelstar** [comment]
Thanks!
